[TTS]在csmsc中添加tts1分支

1726650686 opened the pull request:
<!-- Demo: https://github.com/PaddlePaddle/PaddleNLP/pull/26 -->
### PR types
<!-- One of [ New features | Bug fixes | Function optimization | Performance optimization | Breaking changes | Others ] -->

### PR changes
<!-- One of [ Models | APIs | Docs | Others ] -->

### Describe
<!-- Describe what this PR does -->
1. 提取特征的参数和 csmsc/fastspeech2 等对齐，训练一个 24k 的模型
2. 数据预处理方式和 csmsc/tacotron2 对齐，新增一个 preprocess_new.py
3. 按照t2s/exps/ synthesize.py 和 synthesize_e2e.py做了部分对齐

THUzyt21 committed:
Update pretrained_models.py

Add a new model for faster text process

THUzyt21 committed:
Update model_alias.py

Add a new model for faster text process in cli

THUzyt21 committed:
Update infer.py

change the infer in order to implement the new faster model for text

THUzyt21 committed:
Update README_cn.md

THUzyt21 committed:
Update README.md

THUzyt21 committed:
Update infer.py

add a new faster model to infer in cli

THUzyt21 committed:
Update pretrained_models.py

THUzyt21 committed:
Update README.md

THUzyt21 committed:
Update README.md

THUzyt21 committed:
Update README.md

THUzyt21 committed:
Update README.md

THUzyt21 committed:
precomited

Commit:
训练了transformer声源模型，可与其他声码器匹配

Commit:
训练了transformer声源模型，可与其他声码器匹配

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/PaddlePaddle/PaddleSpeech?pullRequest=2449) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/PaddlePaddle/PaddleSpeech?pullRequest=2449) before we can accept your contribution.<br/>**6** out of **7** committers have signed the CLA.<br/><br/>:white_check_mark: iftaken<br/>:white_check_mark: yt605155624<br/>:white_check_mark: THUzyt21<br/>:white_check_mark: lym0302<br/>:white_check_mark: ZapBird<br/>:white_check_mark: Zth9730<br/>:x: 吕志轩<br/><hr/>**吕志轩** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/PaddlePaddle/PaddleSpeech?pullRequest=2449) it.</sub>

yt605155624 commented on examples/csmsc/tts1/local/synthesize_e2e.sh:
```diff
@@ -0,0 +1,109 @@
+    FLAGS_allocator_strategy=naive_best_fit \
+    FLAGS_fraction_of_gpu_memory_to_use=0.01 \
+    python3 ${BIN_DIR}/../synthesize_e2e.py \
+        --transformer-tts-config=${config_path} \
```
可以看下  ${BIN_DIR}/../synthesize_e2e.py，使用方式不是这样的

iftaken committed:
update readme and fixed bug in ngpu (#2451)

* update readme and fixed ngpu bug

* update png in readme

* update readme and FT web ttsText

lym0302 committed:
fix finetune batch size (#2457)

* fix batch_size, test=tts

THUzyt21 committed:
Update test_cli.sh

update about text cli

THUzyt21 committed:
Update punc_application.yaml

change model

THUzyt21 committed:
Update application.yaml

change model

THUzyt21 committed:
Update text_engine.py

precommihted already

THUzyt21 committed:
Create ernie-3.0.yaml

config file of ernie-3.0-base-zh

THUzyt21 committed:
Rename ernie-3.0.yaml to ernie-3.0-base.yaml

THUzyt21 committed:
Create ernie-3.0-medium-zh

config file of ernie-3.0-medium-zh

THUzyt21 committed:
Rename ernie-3.0-medium-zh to ernie-3.0-medium.yaml

THUzyt21 committed:
Create ernie-3.0-mini.yaml

THUzyt21 committed:
Create ernie-3.0-nano-zh

THUzyt21 committed:
Rename ernie-3.0-nano-zh to ernie-3.0-nano-zh.yaml

THUzyt21 committed:
Create Ernie-tiny.yaml

THUzyt21 committed:
Rename Ernie-tiny.yaml to ernie-tiny.yaml

yt605155624 committed:
add Speaker Diarization in readme, test=doc

yt605155624 committed:
Update README.md

yt605155624 committed:
Update README.md

yt605155624 committed:
Update README_cn.md

yt605155624 committed:
Update README.md

yt605155624 committed:
Update README_cn.md

THUzyt21 committed:
Update text_engine.py

Zth9730 committed:
fix s2t gpu training hang

THUzyt21 committed:
Update text_engine.py

THUzyt21 committed:
Update application.yaml

THUzyt21 committed:
Update punc_application.yaml

THUzyt21 committed:
Update text_engine.py

Commit:
使用transformer的语音合成方式与其他模型对齐

ZapBird committed:
BytesIO类型时，要保证切到初始位置，这样多次读取才能够正常。比如__call__函数。 (#2484)

* BytesIO类型时，要保证切到初始位置，这样多次读取才能够正常。比如__call__函数。
__call__函数的参数audio_file为BytesIO类型时执行到self.preprocess(model, audio_file)会报错，需要判断audio_file为BytesIO类型时执行audio_file.seek(0)。

Commit:
修改
